src/styles/typography/index.md.njk

```diff
@@ -99,6 +99,12 @@ Use the `govuk-link--no-visited-state` modifier class where it is not helpful to
 
 {{ example({group: "styles", item: "typography", example: "link-no-visited-state", html: true, open: true}) }}
 
+### Links without underlines
+
+Links should be underlined by default. Only remove underlines when the number or placement of links makes them difficult to scan.
```

Comment:
> makes them difficult to scan.

How is this determined?

Reply:
I guess this has to be down to the discretion of the designer? You could say something along the lines of 'use underlines unless you observe users struggling to read links'. I think it's quite hard to prove that kind of thing conclusively with user research...

Reply:
Yeah - It's really hard to pinpoint when not to use an underline.

While I'm a strong advocate on putting underlines on links there are a few occasions where I have removed them as they become a bit noisy (in my opinion—I haven't got research to back this up).

![image](https://user-images.githubusercontent.com/37163/75339807-1daf1a00-5889-11ea-8ae1-dde10a9d11b5.png)

In the screenshot of the design system there are no underlines on navigation bars because they are distinguished by their position.

If, however, one of those links weren't links then I think we'd have to add underlines again. For example imagine a header with navigation bar that looked like this

```
Home | About | Welcome Nick Colley | Sign out
```

And ‘Welcome Nick Colley’ isn't a link.

Reply:
Sorry @christopherthomasdesign - I've just realised you've mentioned this in the very next line.